.github/workflows/python-test.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
HI @nilakshi104, Thanks for the PR. You don't have to remove the python versions here. These are used for build test

Reply:
Hi @sivanantha321 , I have updated the file. please review.

Reply:
Thanks ! One small change. Can you update the title,  so it can be more clear what this PR does. For example you can use the issue title. Looking forward for more PRs :)

Reply:
Updated the title. Thanks for pointing that out. sure :)